docs/potato_paper.md

```diff
@@ -0,0 +1,111 @@
+# Potato Paper
+
+## Proof-of-Potato
+
+Proof-of-Potato (PoP) is a protocol for running decentralized sharded off-chain state machines.
+PoP is managed by a potato smart contract hosted on a secure blockchain such as Ethereum.
+The potato smart contract manages potato shard ownership and list of potato verifiers.
+Each potato shard runs a blockchain based potato state machine using our novel Proof-of-Tater (PoT) potato consensus algorithm.
+Other potato consensus protocols are possible too.
+We choose PoT because it offers lightweight security, speed, simplicity, and energy-efficient consensus...
+Just like a potato.
+
+### Potato Contract
+
+The potato contract is a potato smart contract.
+ The blockchain hosting the potato contract is called the potato main chain.
+
+
+**Potato Verifiers**:
+ Users aspiring to be potato verifiers can stake in potatoes in the PC to become farmers.
```

Comment:
I guess I haven't really thought on how stake works

Reply:
you have to freeze potatoes in the potato contract to become a potato verifier. Otherwise anyone can be a potato verifier. That's what staking in means.

Reply:
makes sense